### ISSUE TITLE
Domain Randomization Methods?

### ISSUE THREAD

**Issue by BryanBo-Cao** [facebookresearch/habitat-sim#361]
## ❓ Questions and Help

I was wondering if there are domain randomization methods in habitat(changing color, shading, texture, etc.) that we can use directly? Not just a random initial position/orientation for an agent.

Thanks!

**Comment by aclegg3** [comment]
Thanks for the feature request! 👍 

At this time, we are working on a shader re-design which will allow more control over these sort of features (Issue #319). Note however that many datasets provide illumination baked into textures. We do not provide any methods for dynamic modification of these textures at this time. For assets which do not have baked in illumination, we are considering domain randomization approaches for parameters such as light positions, colors, intensities, specularity, etc... 

In short, we absolutely have this use case in mind and will keep you posted about updates.
